### DISCUSSION TITLE
Way to change row style base on condition

### DISCUSSION THREAD

**Discussion by nyankaty** [jxlsteam/jxls#253] [Q&A] [answered]
Pls help me.
I have been pushed a condition value in object properties, now I want to change row gennerated (color, and font) base on this condition var.
![image](https://github.com/jxlsteam/jxls/assets/59227791/30cdb83c-5068-44c9-9992-96671af74b52)

**Comment by SoltauFintel** [comment, marked as answer]
This does not look like a jxls specific question to me. I guess you can use Excel conditional formatting.
